orienteer-war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -5,14 +5,17 @@
 
 	<display-name>orienteer</display-name>
 
-    <listener>
-        <listener-class>org.orienteer.core.service.OrienteerServletContextListener</listener-class>
-    </listener>
+    <!--<listener>-->
+        <!--<listener-class>org.orienteer.core.service.OrienteerServletContextListener</listener-class>-->
+    <!--</listener>-->
+	<!--<filter>-->
+		<!--<filter-name>orienteer</filter-name>-->
+		<!--<filter-class>com.google.inject.servlet.GuiceFilter</filter-class>-->
+	<!--</filter>-->
 	<filter>
 		<filter-name>orienteer</filter-name>
-		<filter-class>com.google.inject.servlet.GuiceFilter</filter-class>
+		<filter-class>org.orienteer.core.service.ReloadFilter</filter-class>
```

Comment:
It seems that archetypes should be changed as well.
And btw: lets call it OrienteerFilter instead of ReloadFilter.